java/ql/src/experimental/Security/CWE/CWE-117/LogInjection.qll

```diff
@@ -0,0 +1,84 @@
+/**
+ * Provides a taint-tracking configuration for reasoning about untrusted user input used in log entries.
+ */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+
+module LogInjection {
+  string logLevel() {
```

Comment:
[JBoss Logging](https://docs.jboss.org/jbosslogging/latest/org/jboss/logging/BasicLogger.html) also has <code><i>...</i>f</code> and <code><i>...</i>v</code> variants.